react/components/Input/index.js

```diff
@@ -1,6 +1,8 @@
 import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 
+import Button from '../Button'
```

Comment:
I think you should import the button into the new component and pass it as `suffix` to Input